cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -137,7 +138,8 @@
   private final SchedulerQueueResolver schedulerQueueResolver;
 
   // Handles that are running, or not yet completely fetched, they have longer timeout
-  private final Cache<QueryHandle, OperationInfo> activeHandleCache;
+  @VisibleForTesting
+  public final Cache<QueryHandle, OperationInfo> activeHandleCache;
```

Comment:
Is there a way to control when a query completes? I don't want to introduce a flaky test.